dhis-2/.vscode/launch.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
do we need to add this config file?

Reply:
ah ok i was trying out Visual Studio, so I guess we can also add the ignore on master